### DISCUSSION TITLE
When you plan to release 2.1.16 ?

### DISCUSSION THREAD

**Discussion by wstryjew** [smallrye/smallrye-open-api#977] [General]
Hello,
are there any plans to release version 2.1.16?

**Comment by Postremus** [comment]
I need a new release also sometime, in the next few weeks - for all the maven changes.

**Comment by phillip-kruger** [comment]
We can do a release. Anything specific you are waiting for ? Is this for quarkus ?

**Comment by wstryjew** [comment]
I'm waiting for new configuration features of maven plugin. I'd like to use "attachArtifacts".

**Comment by phillip-kruger** [comment]
https://github.com/smallrye/smallrye-open-api/releases/tag/2.1.16

**Reply by codespearhead** [reply]
@wstryjew Can you [mark this discussion as answered](https://docs.github.com/en/discussions/managing-discussions-for-your-community/moderating-discussions#marking-a-comment-as-an-answer)?

**Comment by phillip-kruger** [comment]
Pulling it into Quarkus now
